### PR TITLE
100.6 Sample: Find connected features in utility networks

### PR DESCRIPTION
**Pull request by ZackAllen**

### CODE REVIEW

File: src/Forms/Shared/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+                    // Compute how far tapped location is along the edge feature.
+                    if (feature.Geometry is Polyline line)
+                    {
+                        line = GeometryEngine.RemoveZ(line) as Polyline;
+
+                        // Set how far the element is along the edge.
+                        element.FractionAlongEdge = GeometryEngine.FractionAlong(line, e.Location, -1);
+
+                        Status.Text = $"fractionAlongEdge: `{element.FractionAlongEdge}`";
```
Comment (nCastle1) [inline src/Forms/Shared/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.xaml.cs]: How is the fraction along relevant to the trace operation? Does it need to be shown to the user or is this an implementation detail?

File: src/iOS/Xamarin.iOS/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.cs [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.UI.Controls;
+using Esri.ArcGISRuntime.UtilityNetworks;
+using Foundation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
```
Comment (nCastle1) [inline src/iOS/Xamarin.iOS/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.cs]: Watch out for unused usings.

File: src/iOS/Xamarin.iOS/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.cs [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+            {
+                _activityIndicator.StopAnimating();
+            }
+        }
+
+        private async void OnGeoViewTapped(object sender, GeoViewInputEventArgs e)
+        {
+            try
+            {
+                e.Handled = true;
```
Comment (nCastle1) [inline src/iOS/Xamarin.iOS/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.cs]: What's the purpose of this? Consider adding a comment

File: src/iOS/Xamarin.iOS/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.cs [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+                {
+                    element = _utilityNetwork.CreateElement(feature);
+
+                    // Compute how far tapped location is along the edge feature.
+                    if (feature.Geometry is Polyline line)
+                    {
+                        line = GeometryEngine.RemoveZ(line) as Polyline;
+
+                        // Set how far the element is along the edge.
+                        element.FractionAlongEdge = GeometryEngine.FractionAlong(line, e.Location, -1);
```
Comment (nCastle1) [inline src/iOS/Xamarin.iOS/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.cs]: It seems like this controls where on the feature the barrier/ origin is, but in the results, the position of the barrier isn't being considered? Is that a bug or a limitation in the feature? Or are the results correct and we're just not showing where on the feature the exact boundary is?

File: src/Forms/Shared/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.xaml.cs (hunk 1 above)
Comment (ZackAllen) [reply src/Forms/Shared/Samples/Network Analysis/ConnectedTrace/ConnectedTrace.xaml.cs]: FractionAlongEdge is an attribute of the UtilityElement object. We show the user the value set for that attribute. I can reword the status to be more user friendly and appear less like a debug output.

File: src/Android/Xamarin.Android/ArcGISRuntime.Xamarin.Samples.Android.csproj [hunk 5]
```diff
@@ -499,7 +501,7 @@
       <Version>100.6.0</Version>
     </PackageReference>
     <PackageReference Include="Esri.ArcGISRuntime.Xamarin.Android">
-      <Version>100.6.0</Version>
+      <Version>100.6.0-daily2511</Version>
```
Comment (nCastle1) [inline src/Android/Xamarin.Android/ArcGISRuntime.Xamarin.Samples.Android.csproj]: Please revert this before merging

File: src/Forms/Android/ArcGISRuntime.Xamarin.Forms.Android.csproj [hunk 6]
```diff
@@ -101,10 +101,10 @@
       <Version>100.6.0</Version>
     </PackageReference>
     <PackageReference Include="Esri.ArcGISRuntime.Xamarin.Android">
-      <Version>100.6.0</Version>
+      <Version>100.6.0-daily2511</Version>
     </PackageReference>
     <PackageReference Include="Esri.ArcGISRuntime.Xamarin.Forms">
-      <Version>100.6.0</Version>
+      <Version>100.6.0-daily2511</Version>
```
Comment (nCastle1) [inline src/Forms/Android/ArcGISRuntime.Xamarin.Forms.Android.csproj]: Don't forget to revert this

**Pull request merged**